Core.DomainServices/SystemUsage/BuildItSystemUsageOverviewReadModelOnChangesHandler.cs

```diff
@@ -158,6 +165,33 @@ public void Handle(EntityDeletedEvent<DataProcessingRegistration> domainEvent)
             _pendingReadModelUpdateRepository.Add(PendingReadModelUpdate.Create(domainEvent.Entity.Id, PendingReadModelUpdateSourceCategory.ItSystemUsage_DataProcessingRegistration));
         }
 
+        public void Handle(EntityUpdatedEvent<ItInterface> domainEvent)
+        {
+            _pendingReadModelUpdateRepository.Add(PendingReadModelUpdate.Create(domainEvent.Entity.Id, PendingReadModelUpdateSourceCategory.ItSystemUsage_ItInterface));
+        }
+
+        public void Handle(EntityDeletedEvent<ItInterface> domainEvent)
+        {
+            _pendingReadModelUpdateRepository.Add(PendingReadModelUpdate.Create(domainEvent.Entity.Id, PendingReadModelUpdateSourceCategory.ItSystemUsage_ItInterface));
+        }
+
+        public void Handle(ExposingSystemChanged domainEvent)
+        {
+            //Schedule read model update based on ItSystem if the system was the previous exposer of an interface
+            var oldSystem = domainEvent.PreviousSystem;
```

Comment:
Har du brug for den her? Er det ikke bare en EntityUpdated<ItItnerface>?

Reply:
![image](https://user-images.githubusercontent.com/39791946/116047894-db43c180-a674-11eb-94ae-9095cd2e6988.png)
Det var fordi den blev smidt som event i det kode hvor jeg ellers skulle smide et event